scripts/tasks/webpack-resources.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
       ]
     };
 
-    const devtool = 'source-map';
+    const devtool = 'cheap-module-source-map';
```

Comment:
Would this affect the source maps for *ALL* packages including office-ui-fabric-react?